modules/fedimint-mint-tests/tests/tests.rs

```diff
@@ -0,0 +1,41 @@
+use fedimint_core::sats;
+use fedimint_dummy_client::{DummyClientExt, DummyClientGen};
+use fedimint_dummy_common::config::DummyConfigGenParams;
+use fedimint_dummy_server::DummyGen;
+use fedimint_mint_client::{
+    MintClientExt, MintClientGen, ReissueExternalNotesState, SpendOOBState,
+};
+use fedimint_mint_common::config::MintGenParams;
+use fedimint_mint_server::MintGen;
+use fedimint_testing::fixtures::{next, Fixtures, TIMEOUT};
+
+fn fixtures() -> Fixtures {
+    Fixtures::new()
+        .with_primary(0, MintClientGen, MintGen, MintGenParams::default())
+        .with_module(1, DummyClientGen, DummyGen, DummyConfigGenParams::default())
+}
+
+#[tokio::test(flavor = "multi_thread")]
+async fn sends_ecash_out_of_band() -> anyhow::Result<()> {
+    // Print notes for client1
+    let fed = fixtures().new_fed().await;
+    let (client1, client2) = fed.two_clients().await;
+    let (op, outpoint) = client1.print_money(sats(1000)).await?;
+    client1.await_claim_notes(op, outpoint.txid).await?;
```

Comment:
As you mentioned, we shouldn't need both out point and txid here. `print_money` could save all required data in the operation meta so that a high-level update stream (or simpler await fn) can be built for it.